source/utilities.tex

```diff
@@ -5186,18 +5192,26 @@
 \throws Any exception thrown by the selected constructor of \tcode{T}.
 
 \pnum
-\remarks This constructor shall not participate in overload resolution
-unless tcode{is_constructible_v<T, Args...>} is \tcode{true}.
+\remarks
+This constructor shall not participate in overload resolution unless
+\tcode{is_copy_constructible_v<T>} is \tcode{true}
+and \tcode{is_constructible_v<T, Args...>} is \tcode{true}.
```

Comment:
FWIW I think that's the wrong choice, because if we have an Oxford comma then it seems natural to put the line break after the comma, and start the next line with the conjunction. 

It also involves more editing for cases like:

    case one,
    case two,
    case three, and
    case four.

If you want to add a new case between three and four you have to edit an existing line, not just insert a new one:

    case one,
    case two,
    case three, <DEL>and<DEL>
    <INS>case three-point-five, and</INS>
    case four.